tests/storage/linstor/test_linstor_sr.py

```diff
@@ -78,6 +78,17 @@ def test_snapshot(self, vm_on_linstor_sr):
         finally:
             vm.shutdown(verify=True)
 
+    @pytest.mark.small_vm
+    @pytest.mark.big_vm
+    def test_resize_vdi(self, vm_on_linstor_sr):
+        vm = vm_on_linstor_sr
+        if vm.is_running():
+            vm.shutdown(verify=True, force_if_fails=True)
+        logging.info("* VDI Resize started *")
+        for vdi_uuid in vm.vdi_uuids():
+            vm.vdi_resize(vdi_uuid)
+        logging.info("* VDI Resize completed *")
+
```

Comment:
This looks like a test we might want to add to tests of all other SRs (in a separate PR once this one is merged?)

Reply:
Yes, this one is generally applicable to all storage tests that we have.